.circleci/config.yml

```diff
@@ -165,10 +165,6 @@ jobs:
       - checkout
       - restore_cache:
           key: v4-repo-{{ .Environment.CIRCLE_SHA1 }}
-      - run:
```

Comment:
I apologise, I thought testing didn't require CMake, but it looks like pod install is looking for it. We should add the same thing here with a comment.